src/plans/actions/PlanActionsDropdownItems.tsx

```diff
@@ -0,0 +1,141 @@
+import { type FC, useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom-v5-compat';
+import { ArchiveModal } from 'src/modules/Plans/modals/ArchiveModal';
+import { DuplicateModal } from 'src/modules/Plans/modals/DuplicateModal';
+import { PlanCutoverMigrationModal } from 'src/modules/Plans/modals/PlanCutoverMigrationModal';
+import { PlanDeleteModal } from 'src/modules/Plans/modals/PlanDeleteModal';
+import { PlanStartMigrationModal } from 'src/modules/Plans/modals/PlanStartMigrationModal';
+import useGetDeleteAndEditAccessReview from 'src/modules/Providers/hooks/useGetDeleteAndEditAccessReview';
+import { useModal } from 'src/modules/Providers/modals/ModalHOC/ModalHOC';
+import { useForkliftTranslation } from 'src/utils/i18n';
+
+import { PlanModel, type V1beta1Plan } from '@kubev2v/types';
+import { DropdownItem, DropdownList } from '@patternfly/react-core';
+import { getNamespace } from '@utils/crds/common/selectors';
+import { getPlanIsWarm } from '@utils/crds/plans/selectors';
+import { getPlanURL } from '@utils/crds/plans/utils';
+
+import { PlanStatuses } from '../details/components/PlanStatus/utils/types';
+import {
+  canPlanReStart,
+  canPlanStart,
+  getPlanStatus,
+  isPlanArchived,
+  isPlanExecuting,
+} from '../details/components/PlanStatus/utils/utils';
+
+import { getDuplicateDescription, getEditDescription, getStartDescription } from './utils/utils';
+
+type PlanActionsDropdownItemsProps = {
+  plan: V1beta1Plan;
+};
+
+const PlanActionsDropdownItems: FC<PlanActionsDropdownItemsProps> = ({ plan }) => {
+  const { t } = useForkliftTranslation();
+  const { showModal } = useModal();
+  const navigate = useNavigate();
+
+  const { canDelete } = useGetDeleteAndEditAccessReview({
+    model: PlanModel,
+    namespace: getNamespace(plan),
+  });
+
+  const planURL = getPlanURL(plan);
+  const planStatus = getPlanStatus(plan);
+
+  const canStart = canPlanStart(plan);
+  const canReStart = canPlanReStart(plan);
+  const isWarmAndExecuting = getPlanIsWarm(plan) && isPlanExecuting(plan);
+  const isArchived = isPlanArchived(plan);
+  const buttonStartLabel = canReStart ? t('Restart') : t('Start');
+
+  const [isStartItemEnabled, setIsStartItemEnabled] = useState(canStart);
+
+  useEffect(() => {
+    if (canStart) setIsStartItemEnabled(true);
+  }, [canStart]);
```

Comment:
canSTart can run a lot...

Reply:
It's not memorized, and many other arent, so each change of one will trigger all rest as well

Reply:
I'm not really sure why (or how), but this is meant to protect that plans won't run concurrently, @sgratch any clue why it's here and if we can remove this?